### DISCUSSION TITLE
Organisation membership permissions

### DISCUSSION THREAD

**Discussion by suvayu** [TulipaEnergy/TulipaEnergyModel.jl#79] [Questions]
I'm a bit uncomfortable that everyone has "Owner" permissions, and many don't have 2FA enabled.  Could we please reduce the number of people with Owner permissions, and require they have 2FA?  I suggest 2 persons each from TNO & eSC as owner should be sufficient. My suggestions:

- TNO: @clizbe & @datejada
- eSC: @lsoucasse (I invited him) & @abelsiqueira 

everyone else should be simply members.

**Comment by clizbe** [comment]
That's very gallant of you to suggest others be owners and not yourself.
But yes sounds like a good idea.

- How do we implement 2FA?
- I think I know how to switch the owners. Let me try.

**Reply by suvayu** [reply]
Not gallant, more trying to be less responsible :smirk:

**Comment by clizbe** [comment]
Alright I updated everyone's permissions.
@datejada @lsoucasse @abelsiqueira 

In case you haven't yet, please enable 2FA:
Click your profile photo top right > Settings > Password & Authentication
Click the big green button and follow the steps:
![image](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/assets/11889283/8a2d2917-4b57-4351-8b70-7f21035351fd)
